src/main/java/com/conveyal/analysis/models/AnalysisRequest.java

```diff
@@ -271,21 +267,6 @@ public void populateTask (AnalysisWorkerTask task, UserPermissions userPermissio
         }
     }
 
-    private static void checkGridSize (WebMercatorExtents extents) {
```

Comment:
👍 Moved to `WebMercatorExtents`

I did want to check whether this breaks an implicit dependency hierarchy we wanted to maintain -- not relying on "analysis server" classes (in this case, `AnalysisServerException`) in R5 classes.

Reply:
I did hesitate to use these exceptions given the relationship between the packages, but the overriding consideration was keeping the code mostly unchanged. We've had situations in the past where error messages became invisible or more cryptic to end users because of how they are mapped or interpreted as they pass up from the HTTP server to the UI. So to keep changes minimal and avoid such problems I left the types the same.